playground.js

```diff
@@ -0,0 +1,28 @@
+const form = document.getElementById("form");
+
+form.addEventListener("submit", function (e) {
+  e.preventDefault();
+  if (form === null) {
+    console.error("form element not found");
+    return;
+  }
+
+  const fd = new FormData(form);
+
+  const [replacement, lowercase, trim, fallback] = Array.from(fd.values());
+
+  const opts = {};
+
+  if (replacement.length > 0) {
+    opts.replacement = replacement;
+  }
+
+  opts.lower = lowercase !== undefined;
+
+  opts.trim = trim !== undefined;
+
+  opts.fallback = fallback !== undefined;
+
+  const output = slug(document.getElementById("input").value, opts);
+  document.getElementById("slugOutput").innerHTML = output;
```

Comment:
Assigning via `innerHTML` was fine when it was all defaults and the slug was guaranteed to only contain dashes and alphanumeric characters. Now that `replacement` can be set to (for example) `<b>` or even just `<`, the output won't display properly. I *think* using `innerText` should be OK.

Reply:
I agree here. Have pushed a commit to change it to `innerText` here.